apps/console/src/features/identity-providers/components/forms/authenticators/google-authenticator-form.tsx

```diff
@@ -184,24 +199,57 @@ export const GoogleAuthenticatorForm: FunctionComponent<GoogleAuthenticatorFormP
             const meta: CommonAuthenticatorFormFieldMetaInterface = metadata?.properties
                 .find((meta) => meta.key === value.key);
 
+            /**
+            * Parsing string  to boolean only for Google One Tap value
```

Comment:
Format to --> `Parsing string to boolean only for Google One Tap value.`

Reply:
Fixed 9b4b2bc25c47eef914238e8c4592fd165da371c6